dev/plugins/hu.elte.txtuml.xtxtuml/src/hu/elte/txtuml/xtxtuml/formatting2/XtxtUMLFormatter.xtend

```diff
@@ -150,7 +157,7 @@ class XtxtUMLFormatter extends XbaseFormatter {
 	}
 
 	def dispatch void format(TUAttributeOrOperationDeclarationPrefix it, extension IFormattableDocument document) {
-		regionFor.feature(TU_ATTRIBUTE_OR_OPERATION_DECLARATION_PREFIX__VISIBILITY).append[oneSpace];
```

Comment:
> I recommend to prepend the `type` feature with one space here.

Unfortunately, this is not possible. See my previous [comment](https://github.com/ELTE-Soft/txtUML/pull/525#discussion_r126102659) for an explanation.